### ISSUE TITLE
Suggestion: remove font import

### ISSUE THREAD

**Issue by montezume** [BcRikko/NES.css#34]
[ref](https://github.com/BcRikko/NES.css/blob/b52db2fdf1f08a196a22b0b583b8c5572f614fcd/scss/base/variables.scss)

It would be nice if it was the responsibility of the consumer of the library to import the required Goole Font. 

I would suggest leaving out the import, and updating the documentation to reflect the fact that the user should install the font themselves.

**Comment by BcRikko** [comment]
I was thinking the same thing as you, because I'm a Japanese font user, so `Press Start 2P` is not enough👍

**Comment by montezume** [comment]
@IngwiePhoenix sure, but then you remove the ability to load fonts async for users that want to do that. And for users that are already loading other Google fonts on their site, they will have to make two requests instead of one. 

I think the unintended confusion could be mitigated with decent documentation. Something like:

#### Usage

This framework depends on the google font [Press Start 2P](https://fonts.google.com/specimen/Press+Start+2P?selection.family=Press+Start+2P). You can include it on your website like below.

```css
<head>
    <link href="https://fonts.googleapis.com/css?family=Press+Start+2P" rel="stylesheet">
    <link href="path/to/Nes.css" rel="stylesheet">
</head/>
```

**Comment by jimmiehansson** [comment]
> @IngwiePhoenix sure, but then you remove the ability to load fonts async for users that want to do that. And for users that are already loading other Google fonts on their site, they will have to make two requests instead of one.
> 
> I think the unintended confusion could be mitigated with decent documentation. Something like:
> 
> #### Usage
> This framework depends on the google font [Press Start 2P](https://fonts.google.com/specimen/Press+Start+2P?selection.family=Press+Start+2P). You can include it on your website like below.
> 
> ```
> <head>
>     <link href="https://fonts.googleapis.com/css?family=Press+Start+2P" rel="stylesheet">
>     <link href="path/to/Nes.css" rel="stylesheet">
> </head>
> ```

I agree, this makes the documentation more informative and declarative while loosely coupling the distribution of the font from the framework. :+1:

**Comment by tuxun** [comment]
damn it, again google :/

**Comment by montezume** [comment]
I can spin up a PR if there is some consensus here 😄

**Comment by IngwiePhoenix** [comment]
@montezume I think you may :) Go for it.

**Comment by jimmiehansson** [comment]
Looks solid to me :100:
